dataclients/kubernetes/definitions/ingressvalidator.go

```diff
@@ -0,0 +1,58 @@
+package definitions
+
+import (
+	"fmt"
+
+	"github.com/zalando/skipper/eskip"
+)
+
+const (
+	skipperfilterAnnotationKey    = "zalando.org/skipper-filter"
```

Comment:
Should we export those variables from https://github.com/zalando/skipper/blob/master/dataclients/kubernetes/ingress.go#L23-L25 instead of duplication

Reply:
Maybe export it and move to this package (definitions).